pkg/bugtool/bugtool.go

```diff
@@ -498,11 +511,26 @@ func (s *bugtoolInfo) addBpftoolInfo(tarWriter *tar.Writer) {
 
 func (s *bugtoolInfo) addGopsInfo(tarWriter *tar.Writer) {
 	if s.info.GopsAddr == "" {
+		s.multiLog.Info("Skipping gops dump info as daemon is running without gops, use --gops-address to enable gops")
 		return
 	}
-	s.execCmd(tarWriter, "gops.stack", "gops", "stack", s.info.GopsAddr)
-	s.execCmd(tarWriter, "gpos.stats", "gops", "stats", s.info.GopsAddr)
-	s.execCmd(tarWriter, "gops.memstats", "gops", "memstats", s.info.GopsAddr)
+
+	if s.info.GopsPath == "" {
+		s.multiLog.WithField("gops-address", s.info.GopsAddr).Warn("Failed to locate gops. Please install it or specify its path, see 'bugtool --help'")
+		return
+	}
+
+	_, err := os.Stat(s.info.GopsPath)
+	if err != nil {
+		s.multiLog.WithField("gops-address", s.info.GopsAddr).WithError(err).Warn("Failed to locate gops, please install it.")
+		return
+	}
+
+	s.multiLog.WithField("gops-address", s.info.GopsAddr).WithField("gops-path", s.info.GopsPath).Info("Dumping gops information")
+
+	s.execCmd(tarWriter, "gops.stack", s.info.GopsPath, "stack", s.info.GopsAddr)
+	s.execCmd(tarWriter, "gpos.stats", s.info.GopsPath, "stats", s.info.GopsAddr)
```

Comment:
nit: could you also fix the typo here please? gpos -> gops

Reply:
good catch, will do ;-)